numba/typeinfer.py

```diff
@@ -598,6 +599,13 @@ def get_call_signature(self):
 
 
 class TypeVarMap(dict):
+    def copy(self):
```

Comment:
It's dead code.  Removed at https://github.com/numba/numba/pull/2130/commits/923910912ea7a9a4468a15c8111bc940c0668540